module4/preparation/Week3/MaintainingState.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Maintaining State Preparation
+---
+
+## Pre-Work for [Maintaining State](/module4/lessons/Week3/MaintainingState)
+
+<br>
+
+
+### **"HTTP is a Stateless Protocol"**
+
+The statement above is true.  So, what does it mean?
+
+Spend 10 minutes doing some research to determine what makes HTTP stateless.  You can use the following questions to help guide your research:
+* What is 'state' in relation to web applications?
+* How does a server keep track of requests that have been made to it?
+* How do we 'log in' to applications?
+
+In small groups, discuss what you have learned, and be ready to share out!
+
+<section class='instructor-notes' markdown='1'>
+
+Make sure you have enough time to bring students back together to do a whole group share out.  Students need to understand that each http request is a separate activity - the server does not 'remember' prior requests.
```

Comment:
I like this prep a lot! I think this instructor note brings up a good point of if we want to do whole group share out at the end of the prep period or the next day at the start of the lesson. I could see either working well, but good to make clear the plan :)